src/services/review/ReviewRequestService.ts

```diff
@@ -416,7 +398,7 @@ export default class ReviewRequestService {
 
     const { id: reviewRequestId } = possibleReviewRequest
 
-    await this.reviewRequestView.destroy({
+    this.reviewRequestView.destroy({
```

Comment:
just checking, is this removal of `await` intended?

Reply:
oops, i'll revert this! thanks for spot